entries/unwrap.xml

```diff
@@ -4,6 +4,12 @@
   <signature>
     <added>1.4</added>
   </signature>
+  <signature>
+    <added>3.0</added>
+    <argument name="selector" type="String" optional="true">
+      <desc>A selector to check the parent element against. If the parent element does not match the selector, the element won't be unwrapped.</desc>
```

Comment:
There can be multiple elements so I'd change this to "If an element's parent does not" to emphasize that. Otherwise LGTM!